### DISCUSSION TITLE
Umweltzone - Green zone

### DISCUSSION THREAD

**Discussion by MBj1703** [organicmaps/organicmaps#8387] [Ideas]
Hi,

would be great, if you can add Green Zones and the possibility to avoid them (my Mobilehome is too old to enter them 🥺)

**Comment by biodranik** [comment]
Are green zones already properly mapped in OSM? @map-per can you please help or tag someone who can make a detailed issue on how to get this data from OSM?

Is this feature already implemented in other apps? How does it work there?

**Reply by map-per** [reply]
For low emission zones there is the tag `boundary=low_emission_zone` (393 times used). I looked at a few of these `low_emission_zones` and the access restrictions are often not further specified. In cases where access is mapped (e.g. `motor_vehicle:conditional = no @ (fuel=diesel AND emissions<euro_3)` it's probably difficult to do a mapping from the emission zone area to the affected `highways`.

So displaying any low emission zone should work, but special access conditions for routing probably won't work.

What will be difficult though is to display these areas, that are mostly in dense cities, in a non intrusive way (e.g. patterns like for natural reserves would be to intrusive).

Wiki: https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dlow_emission_zone
All areas worldwide: https://overpass-turbo.eu/?w=%22boundary%22%3D%22low_emission_zone%22+global&R

**Reply by MBj1703** [reply]
The best would be just a "yes or no", so a button to disallow emission zones. Everybody should know his/her car, if it is okay, or not.

**Reply by FedericoCeratto** [reply]
@MBj1703 LEZ can have very complex rules depending on the time of day, `fuel`, `emissions`, etc  (and some even change dynamically based on pollution levels in Italy).
Perhaps users could set `fuel` and `emissions` to have automated go/no-go filtering, plus an override/block flag for each LEZ to customize it as needed.

**Reply by MBj1703** [reply]
I think for most people like me, it would be okay to have a on/off, avoid them or not. Undepending from fuel, emission level, etc.

**Comment by MBj1703** [comment]
I know from some navigation apps like coPilot.
You can select "avoid green zones" (like avoid highway etc.)

**Comment by MBj1703** [comment]
OsmAnd can do it: https://osmand.net
